speech/google/cloud/speech/sample.py

```diff
@@ -69,6 +71,15 @@ def __init__(self, content=None, source_uri=None,
             raise ValueError('Invalid encoding: %s' % (encoding,))
 
     @property
+    def chunk_size(self):
+        """Chunk size to send over GRPC. ~100ms
+
+        :rtype: int
+        :returns: Optimized chunk size.
+        """
+        return int(self.sample_rate / 10)
```

Comment:
I'm fine with the cast, I was at issue with the `/` vs. `//`. Just divide by `10.0` instead of by `10`